src/main/java/edu/oregonstate/cs361/battleship/BattleshipModelNormal.java

```diff
@@ -44,19 +43,15 @@ public BattleshipModelNormal() {
         computerHits = new ArrayList<Point>();
         computerMisses = new ArrayList<Point>();
 
-        playerShips = new ArrayList<Ship>();
-        playerShips.add(playerAircraftCarrier);
-        playerShips.add(playerBattleship);
-        playerShips.add(playerCruiser);
-        playerShips.add(playerDestroyer);
-        playerShips.add(playerSubmarine);
-
-        computerShips = new ArrayList<Ship>();
-        computerShips.add(computerAircraftCarrier);
-        computerShips.add(computerBattleship);
-        computerShips.add(computerCruiser);
-        computerShips.add(computerDestroyer);
-        computerShips.add(computerSubmarine);
+        playerArray[1] = playerBattleship;
+        playerArray[2] = playerSubmarine;
+        playerArray[3] = playerCruiser;
+        playerArray[4] = playerDestroyer;
+
+        computerArray[1] = computerBattleship;
+        computerArray[2] = computerSubmarine;
+        computerArray[3] = computerCruiser;
+        computerArray[4] = computerDestroyer;
```

Comment:
I have to disagree with you on this because there are only x amount of ships so if you don't get that number then you did something wrong. Another reason is that when you create this list dynamically then you would be creating them just like this anyways. I could be misunderstanding you here as well.